config/locales/de.yml

```diff
@@ -15,15 +15,16 @@ de:
       submit: Absenden
 
   message_mailer:
+    username: 50survivors Redaktion
     new_message_email:
       subject: Die Redaktion hat eine neue Frage an Sie
 
   save: Speichern
   discard: Änderungen verwerfen
 
   dashboard:
-    title: Willkommen bei 100eyes. Nutzen Sie die Intelligenz der Community.
-    subtitle: Mit 100eyes kannst du 50 Menschen aus deiner Region Fragen stellen.
+    title: Willkommen bei 50survivors. Nutze die Intelligenz der Community.
```

Comment:
I’m not 💯 sure, but if I recall correctly @FrauCsu and @drjakob preferred the web app to be branded 100eyes, while outside communication should be customized absed on the project/newsroom etc.

@drjakob @FrauCsu, can you clarify?

Should this be changed to "Willkommen bei 50survivors." and "Mit 50survivors kannst du…" or stay as it is right now?
<img width="759" alt="Bildschirmfoto 2020-06-29 um 19 30 21" src="https://user-images.githubusercontent.com/1512805/86037430-a8727200-ba3f-11ea-9521-3b0439a7d976.png">

Same question, should we update this to "50 survivors"?
<img width="382" alt="Bildschirmfoto 2020-06-29 um 19 30 44" src="https://user-images.githubusercontent.com/1512805/86037435-aa3c3580-ba3f-11ea-8c63-6d37c910ad92.png">

Reply:
Its correct.- Webapp should be 100eyes. 50survivors is the name of one research (Recherche). Recherchen are the parents of questions.